### PR TITLE
Add flush() method

### PR DESCRIPTION
**Pull request by nytamin**
Add a `flush()` method, as discussed in #106

### CODE REVIEW

**Comment by nytamin** [comment]
Hey @loucadufault, could you give this PR a quick review to ensure that it fits with your use case (pay extra attention to [the unit tests](https://github.com/SuperFlyTV/xkeys/pull/115/files#diff-85df3dce8b7a01eb511b4e13930d44cc4642e128658f8b8bc6b4e817830f892bR174)) ?

File: packages/node/src/node-hid-wrapper.ts [hunk 1]
```diff
@@ -18,9 +21,13 @@ export class NodeHIDDevice extends EventEmitter implements HIDDevice {
 	}
 
 	public write(data: number[]): void {
-		this.device.write(data).catch((err) => {
-			this.emit('error', err)
-		})
+		this.writeQueue
+			.add(async () => {
+				await this.device.write(data)
+			})
```
Comment (loucadufault) [inline packages/node/src/node-hid-wrapper.ts:27]: small nit: you could directly return the promise here

```suggestion
			.add(() => this.device.write(data))
```

File: packages/node/src/__tests__/xkeys.spec.ts [hunk 2]
```diff
@@ -137,4 +137,95 @@ describe('Unit tests', () => {
...
+		myXkeysPanel.on('error', errorListener)
+
+		mockWriteStart.mockClear()
+		mockWriteEnd.mockClear()
+
+		myXkeysPanel.toggleAllBacklights()
+
+		expect(mockWriteStart).toBeCalledTimes(1)
+		expect(mockWriteEnd).toBeCalledTimes(0) // Should not have been called yet
+
```
Comment (loucadufault) [inline packages/node/src/__tests__/xkeys.spec.ts:203]: you could use jest fake timers instead, and advance the timers here by 10ms to make the test a bit more guaranteed

**Review by loucadufault — APPROVED** [review]
looks to cover the use case proposed in the issue well, thanks!

**Pull request merged**